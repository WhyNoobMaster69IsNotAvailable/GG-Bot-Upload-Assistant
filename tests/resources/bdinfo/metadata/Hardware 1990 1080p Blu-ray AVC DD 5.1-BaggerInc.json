{
    "largest_playlist": "00003.MPLS",
    "raw_video_file": "/data/downloads/FullDisk/Hardware 1990 1080p Blu-ray AVC DD 5.1-BaggerInc/BDMV/STREAM/00003.m2ts",
    "raw_file_name": "Hardware 1990 1080p Blu-ray AVC DD 5.1-BaggerInc",
    "mediainfo": "/app/temp_upload/cd67d76cfa04ecb9540a2fa656b3a87db562bd92b055ecd118dffd15a2fb00cd/mediainfo.txt",
    "upload_media": "/data/downloads/FullDisk/Hardware 1990 1080p Blu-ray AVC DD 5.1-BaggerInc/",
    "bdinfo_output_split": "[\"b'Please\", 'wait', 'while', 'we', 'scan', 'the', 'disc...\\\\n#', 'Group', 'Playlist', 'File', 'Length', 'Estimated', 'Bytes', 'Measured', 'Bytes', '\\\\n\\\\n1', '1', '00003.MPLS', '01:33:53', '18,404,370,432', '-', '\\\\n2', '1', '01003.MPLS', '01:33:53', '18,404,370,432', '-', '\\\\n3', '2', '00004.MPLS', '00:53:59', '7,534,135,296', '-', '\\\\n4', '2', '01004.MPLS', '00:53:59', '7,534,135,296', '-', '\\\\n5', '3', '00005.MPLS', '00:44:30', '7,604,883,456', '-', '\\\\n6', '3', '01005.MPLS', '00:44:30', '7,604,883,456', '-', '\\\\n7', '4', '00009.MPLS', '00:25:02', '4,277,477,376', '-', '\\\\n8', '4', '01009.MPLS', '00:25:02', '4,277,477,376', '-', '\\\\n9', '5', '00006.MPLS', '00:09:50', '1,368,662,016', '-', '\\\\n10', '5', '01006.MPLS', '00:09:50', '1,368,662,016', '-', '\\\\n11', '6', '00007.MPLS', '00:08:33', '1,491,640,320', '-', '\\\\n12', '6', '01007.MPLS', '00:08:33', '1,491,640,320', '-', '\\\\n13', '7', '00008.MPLS', '00:07:40', '1,334,956,032', '-', '\\\\n14', '7', '01008.MPLS', '00:07:40', '1,334,956,032', '-', '\\\\n15', '8', '00011.MPLS', '00:03:30', '510,308,352', '-', '\\\\n16', '8', '01011.MPLS', '00:03:30', '510,308,352', '-', '\\\\n17', '9', '00010.MPLS', '00:02:03', '243,929,088', '-', '\\\\n18', '9', '01010.MPLS', '00:02:03', '243,929,088', '-', '\\\\n19', '10', '00002.MPLS', '00:01:00', '131,303,424', '-', '\\\\n20', '10', '01002.MPLS', '00:01:00', '131,303,424', '-', \"\\\\n'\"]"
}