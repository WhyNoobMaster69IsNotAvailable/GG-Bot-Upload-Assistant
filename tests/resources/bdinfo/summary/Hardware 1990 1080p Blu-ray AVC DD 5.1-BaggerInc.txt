
QUICK SUMMARY:

Disc Title: Hardware
Disc Label: Hardware 1990 1080p Blu-ray AVC DD 5.1-BaggerInc
Disc Size: 42,971,126,887 bytes
Protection: AACS
Playlist: 00003.MPLS
Size: 18,404,370,432 bytes
Length: 1:33:53.461
Total Bitrate: 26.14 Mbps
Video: MPEG-4 AVC Video / 23994 kbps / 1080p / 23.976 fps / 16:9 / High Profile 4.1
Audio: English / Dolby Digital Audio / 5.1 / 48 kHz /   448 kbps / DN -27dB
Audio: English / Dolby Digital Audio / 2.0 / 48 kHz /   256 kbps / DN -27dB
Audio: English / Dolby Digital Audio / 2.0 / 48 kHz /   192 kbps / DN -27dB
