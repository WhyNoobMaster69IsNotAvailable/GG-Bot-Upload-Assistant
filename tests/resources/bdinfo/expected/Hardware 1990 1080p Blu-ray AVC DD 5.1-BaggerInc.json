{
    "video": [
        {
            "codec": "AVC",
            "bitrate": "23994 kbps",
            "resolution": "1080p",
            "fps": "23.976 fps",
            "aspect_ratio": "16:9",
            "profile": "High Profile 4.1"
        }
    ],
    "audio": [
        {
            "language": "English",
            "codec": "Dolby Digital Audio",
            "channels": "5.1",
            "sample_rate": "48 kHz",
            "bitrate": "448 kbps",
            "bit_depth": "DN -27dB"
        },
        {
            "language": "English",
            "codec": "Dolby Digital Audio",
            "channels": "2.0",
            "sample_rate": "48 kHz",
            "bitrate": "256 kbps",
            "bit_depth": "DN -27dB"
        },
        {
            "language": "English",
            "codec": "Dolby Digital Audio",
            "channels": "2.0",
            "sample_rate": "48 kHz",
            "bitrate": "192 kbps",
            "bit_depth": "DN -27dB"
        }
    ],
    "title": "Hardware",
    "label": "Hardware 1990 1080p Blu-ray AVC DD 5.1-BaggerInc",
    "size": 40.01998052652925,
    "playlist": "00003.MPLS",
    "length": "1:33:53"
}